lib/plugins/aws/invokeLocal/index.test.js

```diff
@@ -0,0 +1,1424 @@
+'use strict';
```

Comment:
@Maxgit3 we now keep our tests in `test/unit` and in scope of this PR it's expected that you'll edit [test/unit/lib/plugins/aws/invokeLocal/index.test.js](https://github.com/serverless/serverless/blob/5601025dd8a4075cb463e2dcfb67d6c52984582a/test/unit/lib/plugins/aws/invokeLocal/index.test.js)

Can you update your PR so changes are in `test/unit/lib/plugins/aws/invokeLocal/index.test.js` and so `lib/plugins/aws/invokeLocal/index.test.js` is not re-introduced to the repository?